bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/PageChangeListener.java

```diff
@@ -151,13 +155,25 @@ public void stateChanged(Item item, State oldState, State newState) {
 
     @Override
     public void stateUpdated(Item item, State state) {
+        // For group item only, send an event each time the event state is updated.
+        // It allows updating the group label while the group state is unchanged,
+        // for example the count in label for Group:Switch:OR
+        if (!(item instanceof GroupItem)) {
+            return;
+        }
+        Set<SitemapEvent> events = constructSitemapEvents(item, state, widgets);
+        for (SitemapEvent event : events) {
+            for (SitemapSubscriptionCallback callback : distinctCallbacks) {
+                callback.onEvent(event);
+            }
+        }
     }
 
-    private Set<SitemapEvent> constructSitemapEvents(Item item, State oldState, State newState, List<Widget> widgets) {
```

Comment:
The oldState parameter was not used and I can't provide it in case of "state updated". That is the reason why I suppressed it.
But you're rigjht, the newState parameter is not used too.